### PR TITLE
Backport changes to allow admins to update users' passwords.

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Backport changes to allow admins to update users' passwords (see: https://github.com/canonical-web-and-design/maas-ui/pull/1806).

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- As an admin change a user's password from /settings/user.
- Confirm that the password has changed correctly by logging in as that user with the new password.

## Fixes

https://github.com/canonical-web-and-design/maas-ui/issues/1615.

### CODE REVIEW

File: ui/src/app/settings/views/Scripts/ScriptsList/ScriptsList.test.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ describe("ScriptsList", () => {
 
   it("dispatches action to fetch scripts load", () => {
     const state = { ...initialState };
+    state.scripts.loaded = false;
```
Comment (huwshimi) [inline ui/src/app/settings/views/Scripts/ScriptsList/ScriptsList.test.js:72]: Drive-by fixes for scripts tests.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM!
